### PR TITLE
add gEDA footprint and symbol

### PR DESCRIPTION
**Pull request by t0mpr1c3**

### CODE REVIEW

**Comment by SimonMerrett** [comment]
@t0mpr1c3 how exciting! Would you be kind enough to generate some images to allow me to take a look at it without installing gEDA? Maybe gerber or dxf? I'm not sure what the options are.
Thank you for taking the time to create and share this! I hope others appreciate it too.

**Comment by t0mpr1c3** [comment]
Hi Simon. Yes of course, I can export gerbers and a .png of each side.

It's a nice little project, I just came across the Hackaday article
yesterday and it's perfect for something I am prototyping at the moment.

Best wishes, Tom

**Comment by SimonMerrett** [comment]
Brilliant, I can check gerbers.

Simon

**Comment by SimonMerrett** [comment]
I just checked using kicad gerber viewer. Only query is the diameter of the drills. Seems like about 0.9mm dia whereas SOICbite has been 0.7mm. I think that could even go smaller and escaping between larger diameter holes on a fixed 1.27mm pitch is only going to be harder with a larger hole. What hole size did you use?

File: SOICbite_gEDA.fp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+(
+Pad[75mil -1.100mm 75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "1" ""]
+Pad[25mil -1.100mm 25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "2" ""]
+Pad[-25mil -1.100mm -25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "3" ""]
+Pad[-75mil -1.100mm -75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "4" ""]
+Pad[-75mil -1.100mm -75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "5" "onsolder"]
+Pad[-25mil -1.100mm -25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "6" "onsolder"]
+Pad[25mil -1.100mm 25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "7" "onsolder"]
+Pad[75mil -1.100mm 75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "8" "onsolder"]
+Pin[-100mil -75mil 0.700mm 20mil 0.903mm 0.700mm "GND" "9" "hole"]
```
Comment (SimonMerrett) [inline SOICbite_gEDA.fp]: I can see this is the hole definition. Maybe it's just the colour scheme in my gerber viewer. What does the 0.903mm define and what does the 0.700mm define?

**Comment by t0mpr1c3** [comment]
I specified 0.7 mm drill size in the file, and the gerbers came out at 28
mils which is a tiny bit larger. It's easy enough to change, though.

**Comment by t0mpr1c3** [comment]
That's right. 0.903mm is the solder mask opening. It corresponds to the
drill diameter plus 4 mils on either side.

**Comment by SimonMerrett** [comment]
Then everything looks good for merge - thanks again!

**Comment by t0mpr1c3** [comment]
You're welcome. One last thing -- I left some backup files in the last commit that should not have been there. I raised another PR to delete them.

**Pull request merged**